airlock/settings.py

```diff
@@ -159,8 +159,10 @@ def get_env_var(name):
 
 STATIC_URL = "static/"
 
-ASSETS_DIST = Path(os.environ.get("ASSETS_DIST", BASE_DIR / "assets/dist"))
-STATICFILES_DIRS = [str(ASSETS_DIST)]
+ASSETS_DIST = BASE_DIR / "assets/dist"
```

Comment:
Yeah, I checked with him about this:
https://bennettoxford.slack.com/archives/C069YDR4NCA/p1706006312272769

It turns out we don't need it here because we're trying not to have any build step at all.

Reply:
Yes, but we have no npm build pipeline in airlock (atm) - we import prebuilt static assets from jobs server. So there's no need to configure this at the moment. We can always add it back if needed.